packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5428,16 +5429,31 @@ export default class Meeting extends StatelessWebexPlugin {
           },
         };
 
-        this.cleanupLocalStreams();
-
         this.mediaProperties.setMediaDirection(mediaSettings.mediaDirection);
         this.mediaProperties.unsetRemoteMedia();
 
         // when a move to is intiated by the client , Locus delets the existing media node from the server as soon the DX answers the meeting
-        // once the DX answers we establish connection back the media server with only receiveShare enabled
-        // @ts-ignore - reconnectMedia does not accept any argument
-        await this.reconnectionManager.reconnectMedia(mediaSettings).then(() => {
+        // once the DX answers we close the old connection and create new media server connection with only share enabled
+        const stopStatsAnalyzer = this.statsAnalyzer
+          ? this.statsAnalyzer.stopAnalyzer()
+          : Promise.resolve();
+        await stopStatsAnalyzer
+          .then(() => this.closeRemoteStreams())
+          .then(() => this.closePeerConnections())
+          .then(() => {
+            this.cleanupLocalStreams();
+            this.unsetRemoteStreams();
+            this.unsetPeerConnections();
+            this.reconnectionManager.cleanUp();
+          });
+
+        await this.addMedia({
+          audioEnabled: false,
+          videoEnabled: false,
+          shareVideoEnabled: true,
+        }).then(() => {
```

Comment:
`.then` here is not needed as we're using await already, right?

Reply:
Done